views/peeps.erb

```diff
@@ -0,0 +1 @@
+<%= @peeps %>
```

Comment:
Not finished of course and I guess you would have use the `<% @peeps.each do %>` block here